cadence/nfts/blind-edition-nft/BlindEditionNFT.template.cdc

```diff
@@ -427,6 +432,19 @@ pub contract {{ contractName }}: NonFungibleToken {
 
         admin.link<&{{ contractName }}.Collection{NonFungibleToken.CollectionPublic, {{ contractName }}.{{ contractName }}CollectionPublic, MetadataViews.ResolverCollection}>({{ contractName }}.CollectionPublicPath, target: {{ contractName }}.CollectionStoragePath)
         
+        // Create a queue to hold minted NFTs.
+        //
+        // NFTs are minted into a queue to preserve the mint order.
+        // A CollectionQueue is linked to a collection. All NFTs minted into 
+        // the queue are deposited into the underlying collection.
+        //
+        let queue <- FreshmintQueue.createCollectionQueue(
```

Comment:
> If we are doing a bucket mint, then this resource wont be used right? Since there is no bucket?

Yeah that's right.

> Any particular reason we are doing this here and not just at the mint transaction?

I added it here so that the default collection is also wrapped with a default queue. But I'm realizing that isn't strictly necessary. Do you think its better to just create it lazily in the mint transaction?

Reply:
> Any particular reason we are doing this here and not just at the mint transaction?

I guess one benefit of moving it to the mint transaction is that the contract itself doesn't have to contain any changes. I think I like the idea of decoupling the "mint container" from the contract itself.